builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1269,3 +1325,45 @@ def parseJsonMap(jsonString) {
 	lazyMap.putAll(jsonObj)
 	return lazyMap
 }
+
+def isAzure() {
+  return  service_config['platform'] != null && service_config['platform'] == "azure"
+}
+
+def deleteResourceByTag(env) {
```

Comment:
Function contains business logic related to azure (like populating stackName). Can all of this move to azure module (groovy util)?

Reply:
fixed https://github.com/dev9com/jazz/pull/81/files, will merge once pass travis green